tests/Backend/Workspaces/TeradataWorkspacesUnloadTest.php

```diff
@@ -76,7 +76,7 @@ public function testCreateTableFromWorkspaceWithInvalidColumnNames(): void
             $this->fail('Table should not be created');
         } catch (ClientException $e) {
             $this->assertEquals('storage.invalidColumns', $e->getStringCode());
-            $this->assertStringContainsString('_id', strtolower($e->getMessage())); // RS is case insensitive, others are not
+            $this->assertStringContainsString('_id', $e->getMessage());
```

Comment:
Neni to prilis odvazne? Co na to RS? :-)

Reply:
tady zase umrelo kotatko s copypastem...